packages/ag-charts-community/src/chart/themes/symbols.ts

```diff
@@ -13,3 +15,6 @@ export const DEFAULT_AXIS_GRID_COLOUR = Symbol('default-axis-grid-colour') as un
 export const DEFAULT_BACKGROUND_COLOUR = Symbol('default-background-colour') as unknown as string;
 export const DEFAULT_SHADOW_COLOUR = Symbol('default-shadow-colour') as unknown as string;
 export const DEFAULT_TREEMAP_TILE_BORDER_COLOUR = Symbol('default-treemap-tile-border-colour') as unknown as string;
+
+export const BOLD: FontWeight = 'bold';
+export const BOTTOM: AgChartLegendPosition = 'bottom';
```

Comment:
Why are there constants that aren't symbols in the symbols file?

Reply:
These are additional constants needed by a couple of theme template cases, given these are used in conjunction with in similar places to the symbols above the grouping makes some sense?

Reply:
I've split these into a `constants.ts` file now.